conformance/Makefile

```diff
@@ -2,7 +2,8 @@ NKG_TAG = edge
 NKG_PREFIX = nginx-kubernetes-gateway
 GATEWAY_CLASS = nginx
 SUPPORTED_FEATURES = HTTPRoute,HTTPRouteQueryParamMatching,HTTPRouteMethodMatching,HTTPRoutePortRedirect,HTTPRouteSchemeRedirect,GatewayClassObservedGenerationBump
-KIND_KUBE_CONFIG=$${HOME}/.kube/kind/config
+KIND_NAME ?= kind
```

Comment:
could you document the variable in the table in this readme https://github.com/nginxinc/nginx-kubernetes-gateway/blob/main/conformance/README.md ?